#287 #263  #230 Lifelong Learning Robot Example fixed

abhishek-8081 opened the pull request:
### LFX Mentorship 2025 Term 3: Complete Life-Long Robot Example Implementation for Ianvs

What type of PR is this?
example restoration

What this PR does / why we need it
This PR completes the full implementation of the life-long robot example in the Ianvs project as part of the LFX Mentorship 2025 Term 3.
All major components are complete (example code, tests, documentation). 
The only remaining task is CI/CD integration.

Which issue(s) this PR fixes?
Fixes #287 #263 #230

abhishek-8081 committed:
fullnfinal

Signed-off-by: Abhishek Kumar <abhishekrajputji2004@gmail.com>

hsj576 reviewed (changes requested):
Please clean up the files. **DO NOT** commit the ".github", "build" and other irrelevant files.

abhishek-8081 commented:
> Please clean up the files. **DO NOT** commit the ".github", "build" and other irrelevant files.

alright sir, can you please mention those irrevelant files, although the core directory inside build dir have major changes  that we did in reference to #287

MooreZheng reviewed (changes requested):
We believe there are too many redundant files in this PR (739 update files). Be careful when committing to a global open source community that could affect many members. I agree with all the instructions provided by our Ianvs reviewer, Dr. Shijing @hsj576. 
1.  Only commit those critical and necessary files to lifelong learning. For example, CICD and other examples (revisions owned by Abhishek) should be left to another PR；
2. Make sure to pull the latest ianvs and synchronize files before commit, e.g., the examples/PIPL files (revisions owned by others);
3. @abhishek-8081 might want to take a look at the submitted files yourself, by checking out which file should be uploaded as the other merged PR did. PR authors, instead of reviewers, shall be responsible for the submitted files in his/her own PR.

abhishek-8081 commented:
> We believe there are too many redundant files in this PR (739 update files). Be careful when committing to a global open source community that could affect many members. I agree with all the instructions provided by our Ianvs reviewer, Dr. Shijing @hsj576.
> 
> 1. Only commit those critical and necessary files to lifelong learning. For example, CICD and other examples (revisions owned by Abhishek) should be left to another PR；
> 2. Make sure to pull the latest ianvs and synchronize files before commit, e.g., the examples/PIPL files (revisions owned by others);
> 3. @abhishek-8081 might want to take a look at the submitted files yourself, by checking out which file should be uploaded as the other merged PR did. PR authors, instead of reviewers, shall be responsible for the submitted files in his/her own PR.

Thanks for pointing this out. I’ll review the submitted files, remove unnecessary changes, sync with the latest ianvs, and update the PR accordingly.

abhishek-8081 committed:
Delete .github/ISSUE_TEMPLATE/bug-report.md

abhishek-8081 committed:
Delete .github/ISSUE_TEMPLATE/cleanup.md

abhishek-8081 committed:
Delete .github/ISSUE_TEMPLATE/discussion.md

abhishek-8081 committed:
Delete .github/ISSUE_TEMPLATE/documentation.md

abhishek-8081 commented:
Closing this PR. A revised and properly cleaned PR has been raised at #296  addressing all reviewer feedback. I appreciate the review and request your review on the new PR.
